test/components/ProcessingOrder_spec.js

```diff
@@ -0,0 +1,30 @@
+import { expect } from 'chai';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import TestUtils from 'react-addons-test-utils';
+import constants from '../../src/constants';
+import ProcessingOrderComponent from '../../src/components/ProcessingOrder';
+
+const {
+  renderIntoDocument,
+  findRenderedDOMComponentWithClass
+} = TestUtils;
+const ProcessingOrder = React.createFactory(ProcessingOrderComponent);
+
+describe('ProcessingOrder', () => {
+  it('renders subtotal for all delivered entries', () => {
+    const props = {
+      order: {
+        status: 'open', id: 1, entries: [
+          { name: 'Pho', status: constants.DELIVERED, price: 1025 },
+          { name: 'Rice', status: constants.CANCELED, price: 125 }
```

Comment:
you can use a generator for this since we only really care about the status and price. would make the test more explicit.